src/test/java/codeu/controller/ProfileServletTest.java

```diff
@@ -0,0 +1,40 @@
+package codeu.controller;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+import static org.junit.Assert.*;
+
+public class ProfileServletTest {
+    /*
+    private ProfileServlet profileServlet;
```

Comment:
Can you uncomment/fix these tests?

Reply:
A good start for tests would be to check the message posted (if you have other ideas, that works too!)